.gitignore

```diff
@@ -1,5 +1,23 @@
+# MkDocs build dir
 site/
 
 .terraform
 
 node_modules/
+
+# OS generated files
+.DS_Store
+
+# IDE files
+.vscode
+.metals
+.idea
+
+# Backup files
+*.swp
+*.bak
+*.tmp
+*~
+
+# Auxiliary tool outputs
+tools/*.csv
```

Comment:
This isn't used for now (since we don't have the `tools/` directory on this repository), but it's not a problem if you leave this here.

Reply:
I'll remove it, to be more consistent.